README.md

```diff
@@ -0,0 +1,86 @@
+# Logstash Plugin
+
+This is a plugin for [Logstash](https://github.com/elastic/logstash).
+
+It is fully free and fully open source. The license is Apache 2.0, meaning you are pretty much free to use it however you want in whatever way.
+
+## Documentation
+
+Logstash provides infrastructure to automatically generate documentation for this plugin. We use the asciidoc format to write documentation so any comments in the source code will be first converted into asciidoc and then into html. All plugin documentation are placed under one [central location](http://www.elastic.co/guide/en/logstash/current/).
```

Comment:
Yeah, it got generated when I ran the plugin generator: https://www.elastic.co/guide/en/logstash/current/_how_to_write_a_logstash_filter_plugin.html#_use_the_plugin_generator_tool_3.

I see this text in other plugins' READMEs. Should I keep it here for consistency with other plugins or delete it so it reflects the current state of the world accurately? I think the latter but want to confirm first.

Reply:
I would change the text. You can look at the PRs I have open here: https://github.com/logstash-plugins/logstash-input-example/pull/12

Reply:
@dedemorton @jakelandis Since I generated this `README.md` using the plugin generator (`bin/logstash-plugin generate ...`), I think I should also make the suggested changes in:

* https://github.com/elastic/logstash/blob/master/lib/pluginmanager/templates/input-plugin/README.md#documentation
* https://github.com/elastic/logstash/blob/master/lib/pluginmanager/templates/filter-plugin/README.md#documentation, and
* https://github.com/elastic/logstash/blob/master/lib/pluginmanager/templates/output-plugin/README.md#documentation

Otherwise any future plugin authors will end up with the wrong text. Correct?